tests/integration/drivers/CgreenDriverTest.cpp

```diff
@@ -0,0 +1,63 @@
+#include <cgreen/cgreen.h>
+#include <cucumber-cpp/autodetect.hpp>
+#include "../../utils/DriverTestRunner.hpp"
+
+using namespace cgreen;
+using namespace cucumber;
+using namespace cucumber::internal;
+
+THEN(SUCCEED_MATCHER) {
+    ScenarioScope<SomeContext> ctx;
+    assert_that(true);
+}
+
+THEN(FAIL_MATCHER) {
+    ScenarioScope<SomeContext> ctx;
+    assert_that(false);
+}
+
+THEN(PENDING_MATCHER_1) {
+    pending();
+}
+
+THEN(PENDING_MATCHER_2) {
+    pending(PENDING_DESCRIPTION);
+}
+
+class CgreenStepDouble : public CgreenStep {
+public:
+    bool isInitialized() {
+        return CgreenStep::initialized;
+    }
+
+    const InvokeResult invokeStepBody() {
+        return CgreenStep::invokeStepBody();
+    };
+
+    void body() {};
+};
+
+class CgreenDriverTest : public DriverTest {
+public:
+    virtual void runAllTests() {
+        stepInvocationInitsCgreenTest();
+        DriverTest::runAllTests();
+    }
+private:
+    void stepInvocationInitsCgreenTest() {
+        std::cout << "= Init =" << std::endl;
```

Comment:
Unless you really want .flush() to be called here I suggest you just terminate the string with `"\n"` instead of streaming in `std::endl`. `std::endl` does nothing more than streaming in `"\n"` followed by calling `.flush()`.

FYI: some people think it takes care of converting to the current OS' end-of-line character sequence: it doesn't, that's what the underlying stream itself already does for `"\n"` (exactly like _not_ passing the "b" flag to fopen() does for C, in fact: on GNU's libstdc++ it uses the C library).